client/mysql.cc

```diff
@@ -3499,12 +3499,14 @@ com_help(String *buffer MY_ATTRIBUTE((unused)),
 	  return com_server_help(buffer,line,help_arg);
   }
 
-  put_info("\nFor information about Percona products and services, visit:\n"
+  my_snprintf((char*) glob_buffer.ptr(), glob_buffer.alloced_length(),
+	   "\nFor information about Percona products and services, visit:\n"
            "   http://www.percona.com/\n"
-	   "Percona Server manual: http://www.percona.com/doc/percona-server/5.6\n"
+	   "Percona Server manual: http://www.percona.com/doc/percona-server/%d.%d\n"
```

Comment:
tabs instead of spaces

Reply:
@inikep 
MySQL has coding standards and we are their fork. So we don't have any choice rather than stick to them. Moreover, InnoDB (everything in storage/innobase/*) has different standards than general SQL (everything else).

The most noticeable differences:
- InnoDB uses tabs for indentation: 1 tab == 8 spaces.
- Innodb has standard block indentation of 1 tab.
- Innodb has opening curly braces { at the same line (for instance `if (cond) {`).
- InnoDB has spaces before assignment operators/initialization (`int x = 8;`).
- InnoDB does not like single line comments `// my comment`.
- SQL uses spaces for indentation.
- SQL has standard block indentation of 2 spaces.
- SQL has opening curly braces `{` at a new line.
- SQL has no spaces before assignment operators/initialization (`int x= 8;`).
- SQL does not like single line comments `// my comment` although you may encounter them occasionally.
- Both InnoDB and SQL have max line width == 78 characters (80 - 2 for EOL on Windows).

Everything else can be found in the code